docs/ValidationRules/RULEID.RULEFRIENDLYNAME.cs

```diff
@@ -0,0 +1,142 @@
+ *          ReferToFinalSchema
+ *
+ * - Rename this file as <RULEID>.<RULEFRIENDLYNAME>.cs.
+ * 
+ * - Remove All INSTRUCTIONS after the changes have been made.
+ */
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class RULEFRIENDLYNAME : SarifValidationSkimmerBase
+    {
+        private readonly MultiformatMessageString _fullDescription = new MultiformatMessageString
+        {
+            /*
+             * INSTRUCTIONS:
+             * Add a new key-value pair in ~\Sarif.Multitool\Rules\RuleResources.resx.
+             * 
+             * Key: 
+             *      RULEID_RULEFRIENDLYNAME
+             *      
+             * Value: 
+             *      Provide a brief description with atleast two sentences, both ending in a period.
+             *      The first sentence should be a short description of the rule.
+             *      The second sentence must describe the utility/usage of the rule.
+             *      
+             * Example:
+             *      The $schema property should be present, and must refer to the final version 
+             *      of the SARIF 2.1.0 schema. This enables IDEs to provide Intellisense for SARIF log files.
+             *      
+             * Notes:
+             *      The first sentence will be used as a `ShortDescription` for this rule.
+             *      All sentences together will be used as a `LongDescription` for this rule.
+             */
+            Text = RuleResources.RULEID_RULEFRIENDLYNAME_Description
+        };
+
+        public override MultiformatMessageString FullDescription => _fullDescription;
+
+        /*
+         * INSTRUCTIONS:
+         * Decide the appropriate FailureLevel for this rule as appropriate.
+         * The following heuristics can be used to arrive at a decision:
+         * 
+         * Error:
+         *      In general, an `Error` should be reserved for rules which address
+         *      a SARIF spec violation that cannot be expressed by the Schema.
+         *      Example:
+         *      SARIF1019.RuleIdMustBePresentAndConsistent
+         *      Per spec, at least one of result.ruleId and result.rule.id must be present, 
+         *      and if both are present, they must be the same.
+         *      
+         * Warning:
+         *      In general, a `Warning` is a good practice which should be followed, but it
+         *      does not violate the SARIF spec.
+         *      Example:
+         *      SARIF1021.DoNotUseFriendlyNameAsRuleId
+         */
+        public override FailureLevel DefaultLevel => FailureLevel.Warning;
+
+
+        /*
+         * INSTRUCTIONS:
+         * Add a new property in ~\src\Sarif.Multitool\Rules\RuleId.cs with 
+         * the name as RULEFRIENDLYNAME and 
+         * the value as RULEID.
+         * 
+         * Example:
+         *      public const string ReferToFinalSchema = "SARIF1020";
+         */
+        /// <summary>
+        /// RULEID
+        /// </summary>
```

Comment:
Despite the fact that I have slavishly copied it into every rule I've written, I don't think this comment has any value.